.github/workflows/continuous-integration-workflow.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, windows-2019]
-        python-version: ['3.8', '3.9', '3.11']
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
If so, test/example are not run on the 3.8. Can we just test all 4 versions? If it's not able to, we can choose 3.8 (pipeline version), 3.10 (dev version), 3.11 (next version).

Reply:
We can test all, but this is redundant with coverage so we shouldn't include the version we are using for coverage